selfdrive/car/honda/values.py

```diff
@@ -1441,6 +1441,12 @@ def init_make(self, CP: car.CarParams):
       b'78108-T21-A230\x00\x00',
       b'78108-T22-A020\x00\x00',
     ],
+    (Ecu.fwdRadar, 0x18dab0f1, None): [
```

Comment:
This is actually the fwdCamera, since this car doesn't have a radar. The problem is this address is used for both radar and camera in different vehicles, and we have a test to assert consistent addrs to ecus